types/three/three-core.d.ts

```diff
@@ -1606,6 +1606,18 @@ export class Object3D extends EventDispatcher {
     userData: any;
 
     /**
+     * Calls before rendering object
+     */
+    onBeforeRender: (renderer: THREE.WebGLRenderer, scene: THREE.Scene, camera: THREE.Camera, geometry: THREE.Geometry | THREE.BufferGeometry,
+                     material: THREE.Material, group: THREE.Group) => any;
```

Comment:
I think this and `onAfterRender` should return void rather than any. Saying they return any implied to me that there was potentially some meaningful return value that would inform the caller of something, but we just had not specified it's shape yet.